### ISSUE TITLE
is there any interception for windows11

### ISSUE THREAD

**Issue by Lol4ik1284** [oblitum/Interception#149]
I would like to know if there is di intercept for window11 because when I tried to use these files on window 11, it didn’t work out for me and the window wrote that a newer version was needed

**Comment by maot01** [comment]
Would like to know as well 🙂

**Comment by joaomcarlos** [comment]
I am using the driver on Win11. It works. If you dont have virtualization services turned on on your motherboard it may not work (I dont remember if there was an extra security setting for that, I saw a post about it sometime ago when I had the same problem).

I have a problem where from time to time it gets auto-removed and I have to run the /uninstall command from a admin elevated command line, restart the computer, run the install from admin elevated command line and restart again.

After some time, it stops working again. I havent managed to understand what is the condition that makes it get removed.

**Comment by maot01** [comment]
Humm good that it works if that is toggled on at least, but sounds like Microsoft is screwing us with the above mentioned toggling on / off at random.

**Comment by SuperMarioYAHOOYAHOO** [comment]
yes, i installed interception on my windows 11 pc.  windows 11 has been supported as in 2023.

**Comment by maot01** [comment]
> virtualization

And what about virtualization?

**Comment by maot01** [comment]
> yes, i installed interception on my windows 11 pc. windows 11 has been supported as in 2023.

And what about virtualization?

**Comment by SuperMarioYAHOOYAHOO** [comment]
still works
________________________________

**Comment by SuperMarioYAHOOYAHOO** [comment]
works
________________________________

**Comment by SuperMarioYAHOOYAHOO** [comment]
> > yes, i installed interception on my windows 11 pc. windows 11 has been supported as in 2023.
> 
> And what about virtualization?

it worked for me
